examples/cpp/common/opengl/GlUtils.cpp

```diff
@@ -562,6 +562,7 @@ namespace oria {
     mv.withPush([&]{
       mv.rotate(-HALF_PI - 0.22f, Vectors::X_AXIS).scale(0.5f);
       renderGeometry(shape, program, [&] {
+        Uniform<float>(*program, "ForceAlpha").Set(alpha);
```

Comment:
You need to change this so that ForceAlpha is only set if the incoming value is not 0.  Actually it would be better if the default parameter defined in the header was -1 and then only set alpha if it's >= 0